siad/daemon.go

```diff
@@ -202,31 +237,28 @@ func startDaemon(config Config) (err error) {
 		tpool,
 		w,
 	)
-	if err != nil {
-		return err
-	}
 
-	// Bootstrap to the network.
-	if !config.Siad.NoBootstrap && g != nil {
-		// connect to 3 random bootstrap nodes
-		perm, err := crypto.Perm(len(modules.BootstrapPeers))
-		if err != nil {
-			return err
-		}
-		for _, i := range perm[:3] {
-			go g.Connect(modules.BootstrapPeers[i])
-		}
-	}
+	// connect the API to the server
+	srv.mux.Handle("/", a)
```

Comment:
it's fine to leave that to another PR just trying to make sure I'm not missing anything.